sunkit_instruments/iris/obsid.py

```diff
@@ -22,8 +24,7 @@ class ObsId(dict):
     Examples
     --------
     Quickly show OBS ID parameters:
-
-    >>> obsid.ObsId(3675508564)
+    >>> obsid.ObsId(3675508564)  # doctest: +SKIP
```

Comment:
Since we've already merged a PR fixing up ```ObsId```, these doctests should not be skipped and should pass.

Reply:
@DanRyanIrish I don't think we have merged the `ObsId` PR yet if you are referring to https://github.com/sunpy/sunkit-instruments/pull/18. But seems like some import statements are missing. Would you like me to add these in?